todo/src/reducers/index.js

```diff
@@ -0,0 +1,42 @@
+export const ADD_TASK = 'ADD_TASK';
+export const TOGGLE_COMPLETE = 'TOGGLE_COMPLETE'
+export const CLEAR_COMPLETED = 'CLEAR_COMPLETED'
+
+const initialState = {
+	todoList: [
```

Comment:
You could set some initial state here just to have a few items appear so the user knows how to interact w/ the todo list.